config/owasp/suppressions.xml

```diff
@@ -157,16 +157,11 @@
     </suppress>
     <suppress>
         <notes><![CDATA[
-       file name: gradle-pitest-plugin-1.3.0.jar
-       ]]></notes>
-        <sha1>544d22a32e5c409c643a6a7f15513f74bbabccde</sha1>
+   file name: gradle-pitest-plugin-1.4.0.jar
```

Comment:
do't we have any alternative rather suppressing?

Reply:
Afraid not, this is the latest version that works with Java 11 according to the PITest website, seems the vulnerability occurs in all their versions

Reply:
this has been addressed here: https://github.com/hmcts/rd-professional-api/pull/615/commits/f6da97dfbdd2211f0e852654fa3073051ed08707